ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -674,6 +674,13 @@ public class StoreConfig {
   public final boolean storeBlockStaleBlobStoreToStart;
   public final static String storeBlockStaleBlobStoreToStartName = "store.block.stale.blob.store.to.start";
 
+
+  @Config(ENABLE_FILE_COPY_FOR_BOOTSTRAP)
+  @Default("false")
+  public final boolean enableFileCopyForBootstrap;
+  public static final String ENABLE_FILE_COPY_FOR_BOOTSTRAP = "clustermap.enable.file.copy.for.bootstrap";
```

Comment:
Either this or the the key defined in ClusterMapConfig needs to be removed.
Further, 
1. Key in this file will have `store` as prefix
2. Keeping the nomenclature in this file consistent, the key name would be `storeEnableFileCopyForBootstrap`

Reply:
Noted.